recipes/vim/meta.yaml

```
@@ -0,0 +1,50 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 13 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "vim" %}
+{% set version = "8.1.0925" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/vim/vim/archive/v{{ version }}.tar.gz
+  sha256: 6197407497c5cac832b4417ece6ed4f54c03336ae947130e2ad295e480f0fb5f
+
+build:
+  skip: true  # [win]
+  number: 0
+
+requirements:
+  build:
+    # if your project compiles code (such as a C extension) then add the required compilers as separate entries here.
+    # compilers are named 'c', 'cxx' and 'fortran'.
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - make
+  host:
+    - python
+    - ncurses
+  run:
+    - python
+    - ncurses
+
+test:
+  commands:
+    - vim --version
+
+about:
+  home: https://www.vim.org/
+  # vi(m) is licensed as charity-ware
+  license: OTHER
+  # license_file: LICENSE.txt
```

So far as I can tell the license isn't being packaged, I think you can find it here:
```suggestion
  license_file: runtime/doc/uganda.txt
```